openshift/templates/api-deployment-config.yaml

```diff
@@ -0,0 +1,168 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: ${CONTAINER_NAME}-template
+labels:
+  template: ${CONTAINER_NAME}-template
+
+objects:
+  - kind: DeploymentConfig
+    apiVersion: apps.openshift.io/v1
+    metadata:
+      annotations:
+        app.openshift.io/route-disabled: 'false'
+        openshift.io/generated-by: OpenShiftWebConsole
+      name: ${CONTAINER_NAME}
+      namespace: ${OPENSHIFT_PROJECT}
+      labels:
+        app: ${CONTAINER_NAME}
+        app.kubernetes.io/component: ${CONTAINER_NAME}
+        app.kubernetes.io/instance: ${CONTAINER_NAME}
+        app.kubernetes.io/name: ${CONTAINER_NAME}
+        app.kubernetes.io/part-of: ${APPLICATION_NAME}
+        app.openshift.io/runtime: nodejs
+        app.openshift.io/runtime-namespace: ${OPENSHIFT_PROJECT}
+    spec:
+      strategy:
+        type: Rolling
+        rollingParams:
+          updatePeriodSeconds: 1
+          intervalSeconds: 1
+          timeoutSeconds: 600
+          maxUnavailable: 25%
+          maxSurge: 25%
+        resources: {}
+        activeDeadlineSeconds: 21600
+      triggers:
+        - type: ImageChange
+          imageChangeParams:
+            automatic: true
+            containerNames:
+              - ${CONTAINER_NAME}
+            from:
+              kind: ImageStreamTag
+              namespace: ${OPENSHIFT_PROJECT}
+              name: '${CONTAINER_NAME}:latest'
+            lastTriggeredImage: >-
+              ${IMAGE_REPOSITORY}/${OPENSHIFT_PROJECT}/${CONTAINER_NAME}
+        - type: ConfigChange
+      replicas: 1
+      revisionHistoryLimit: 10
+      test: false
+      selector:
+        app: ${CONTAINER_NAME}
+        deploymentconfig: ${CONTAINER_NAME}
+      template:
+        metadata:
+          creationTimestamp: null
+          labels:
+            app: ${CONTAINER_NAME}
+            deploymentconfig: ${CONTAINER_NAME}
+          annotations:
+            openshift.io/generated-by: OpenShiftWebConsole
+        spec:
+          containers:
+            - name: ${CONTAINER_NAME}
+              image: >-
+                ${IMAGE_REPOSITORY}/${OPENSHIFT_PROJECT}/${CONTAINER_NAME}
```

Comment:
we're sure that this equates to artifacts.developer.gov.bc.ca/cec1-ec1236-image-repository/spr-api:latest

Reply:
I've updated this to better reflect that path. I think the artifactory bit was missing.